Sources/Identity/CustomerInfoManager.swift

```diff
@@ -41,7 +42,7 @@ class CustomerInfoManager {
                                      withRandomDelay: isAppBackgrounded) { result in
             switch result {
             case let .failure(error):
-                self.deviceCache.clearCustomerInfoCacheTimestamp(appUserID: appUserID)
+                self.withData { $0.deviceCache.clearCustomerInfoCacheTimestamp(appUserID: appUserID) }
```

Comment:
Hmm I believe in this case the behavior would be the same and considering we are not modifying data in the `Data` struct, this is correct, but considering this is clearing a cache, would it semantically be better to use `self.modifyData`?

Reply:
In terms of naming semantics I agree, but then we'd have to ignore this `inout` parameter which I think would make it more confusing.